handlers/productsHandler.js

```diff
@@ -0,0 +1,57 @@
+const Product = require('../model/productsModel');
+
+
+const allProducts = async (req,res) => {
+	const productsList = await Product.find();
+
+	if(!productsList) {
+		return res.status(500).json({success:false})
+	}
```

Comment:
```suggestion

	
```

Same reason as previous codes, please handle unexpected rejections in a try and catch clause.